.github/workflows/create-release.yaml

```diff
@@ -0,0 +1,34 @@
+name: Create release
+
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  ci:
+    uses: ./.github/workflows/waspc-ci.yaml
+
+  build:
+    uses: ./.github/workflows/waspc-build.yaml
+
+  create-release:
+    needs: [ci, build]
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          pattern: wasp-*
```

Comment:
I am curious, what if for example there are some old aftifacts up there, and they are also starting with `wasp-`? Would it pick up those also, even though we probably didn't want that to happen?

Not sure if this is real problem, but some quick ideas to possibly avoid this if it is (they might not be doable, I am making them up):
1. Pass version tag as input to build job, so that artifacts are marked with that someohw (in the name, or they are in such dir?), and then we reference them that way also.
2. Or maybe have build job return exact names of the artifacts it created (I don't konw if that is possible, if jobs can return outputs) and then supply those here.